lib/evm/src/client.rs

```diff
@@ -46,4 +46,12 @@ impl EVMClient for BasinClient {
             .map_err(|e| Error::Evm(e.to_string()))?;
         Ok(())
     }
+
+    async fn list_pub(&self, owner: Address) -> Result<Vec<String>, Error> {
```

Comment:
Implementation of the [method of the trait](https://github.com/tablelandnetwork/basin-provider/pull/9/files#diff-754f0b45540b51df93364c3267dd745bc15db3352ddc429b4b8183b5e1d3b351R22)